libr/bin/bin.c

```diff
@@ -168,6 +168,17 @@ R_API void r_bin_info_free(RBinInfo *rb) {
 	free (rb);
 }
 
+R_API RBinImport *r_bin_import_clone(RBinImport *o) {
+	RBinImport *res = r_mem_dup (o, sizeof (*o));
+	if (!res) {
+		return NULL;
+	}
+	res->name = o->name ? strdup (o->name) : NULL;
```

Comment:
use r_str_dup, which already does this nullchk for you, or maybe better use a macro

Reply:
kk